test/test_mps.py

```diff
@@ -6931,6 +6931,14 @@ def compare_mm(m, n, k):
         # see https://github.com/pytorch/pytorch/issues/116769#issuecomment-1920066984
         compare_mm(32769, 1, 1025)
 
+        if product_version >= 14.0:
```

Comment:
Can we run this with opinfo to get a more diverse set of inputs?

Reply:
What are these tests covering that the OpInfo is not?

Reply:
Is there any concerns for test-times matrix sizes anymore? If not, I can just add this matrix sizes to OpInfo and be done with it, but it will probably hit some ASAN timeouts